python/pyarrow/tests/test_acero.py

```diff
@@ -19,18 +19,9 @@
 
 import pyarrow as pa
 import pyarrow.compute as pc
+import pyarrow.acero as ac
 from pyarrow.compute import field
 
-from pyarrow._acero import (
```

Comment:
I would personally prefer that you revert the changes to this PR, and instead update this import to `from pyarrow.acero import ( ...` (that avoids unnecessary changes reducing the diff)

Reply:
done